glibc-2.3.3-1478-g37582bc

* Thu Nov 30 2006 Jakub Jelinek <jakub@redhat.com> 2.5.90-9
- on ppc64 build __libc_start_main without unwind info,
  as it breaks MD_FROB_UPDATE_CONTEXT (#217729, #217775; in the
  future that could be fixable just by providing .cfi_undefined r2
  in __libc_start_main instead)

diff -Nru glibc-2.17-c758a686/sysdeps/powerpc/powerpc64/Makefile glibc-2.17-c758a686/sysdeps/powerpc/powerpc64/Makefile
--- glibc-2.17-c758a686/sysdeps/powerpc/powerpc64/Makefile	2012-06-05 07:42:49.000000000 -0600
+++ glibc-2.17-c758a686/sysdeps/powerpc/powerpc64/Makefile	2012-06-07 12:15:21.828318633 -0600
@@ -35,6 +35,8 @@ CFLAGS-rtld-memmove.os = $(no-special-re
 CFLAGS-rtld-memchr.os = $(no-special-regs)
 CFLAGS-rtld-strnlen.os = $(no-special-regs)
 
+CFLAGS-libc-start.c += -fno-asynchronous-unwind-tables
+
 ifeq ($(subdir),csu)
 sysdep_routines += hp-timing
 elide-routines.os += hp-timing
